src/Http/Client.php

```diff
@@ -227,6 +227,31 @@ public function addCookie(CookieInterface $cookie)
         return $this;
     }
 
+    /**
+     * Pass a string url to set the base domain and scheme for the request.
+     * Url must parse into a scheme and host.
+     *
+     * @param  string  $url A string url e.g. https://example.com
```

Comment:
There are some extra white space in there that shouldn't be.
Also, in text URL is usually used as acronym.